src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx

```diff
@@ -1,11 +1,17 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable react/no-unstable-nested-components */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable react/react-in-jsx-scope */
 import { useSelector } from 'react-redux';
 import { PieChart } from 'components/common/PieChart';
-import { peopleTasksPieChartViewData } from '../selectors';
 import { ReportPage } from 'components/Reports/sharedComponents/ReportPage';
 import { NewModal } from 'components/common/NewModal';
+import { peopleTasksPieChartViewData } from '../selectors';
 import './PeopleTasksPieChart.css';
 
-export const PeopleTasksPieChart = () => {
+export function PeopleTasksPieChart() {
```

Comment:
I am sure it works fine but why did you change from arrow function to normal?

Reply:
<img width="505" alt="image" src="https://user-images.githubusercontent.com/67836976/211168848-285a243b-849a-4d3d-97cb-58fc724600db.png">

Reply:
Hi Lee, I think it was those auto-fix. because I usually use the arrow function myself, I don't think I changed it intentionally.

Reply:
This chart's values are wrong because they are hard coded. I didn't connect the data with the pie chart. I talked with Jae and a full-stack would take care of it because it has to do with the data from the database as well and I still can't do the backend.